### DISCUSSION TITLE
2024-08-08 - Separating urges from negative behavior - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#8999] [Chewing the Fat]
https://www.ssw.com.au/rules/separate-urge-from-behavior

Any stories from when you were frustrated and reacted immediately?

Any other tips to avoid immediately reacting?

**Comment by JeanThirion** [comment]
Great technique - another one is to "walk out of the room" when you start feeling angry.
Takes a bit of practice to feel your emotions before they surface, but it's very efficient too

**Comment by PothieuG** [comment]
It's a good behavior reminder.
Acting angry can make us say or do things we might regrets

**Comment by Freego1783** [comment]
I'm already doing something similar when I start feeling angry, espcially in my personal life. I think I'll try to apply this when I feel like getting a snack 😬

**Comment by tiagov8** [comment]
Basically anytime you're emotionally affected your behaviour won't be ideal. So waiting until you're feeling calmer is a good practice.

--

I've often think about the example where if you get honked at in traffic...

...after getting a parking ticket in an extremely hot day with the A/C broken...
You might get really upset 🤬

Now if you get the exact same honk...

...after being promoted and are listening to your favorite song... 
You might don't even think about it ✌️

It's not the honk, but your state of mind that makes the difference.

**Reply by jakebayliss** [reply]
Love this

**Comment by bahjatalaadelssw** [comment]
Leaving the room is a good technique. Islam teaches: Sit down if you're standing, if that doesn't help then lie down. Also wash your face with cool water.

**Comment by leahy268** [comment]
Walking away is the simplest thing.  
Being a SysAdmin you often run into people that frustrate you because they haven't bothered to check if something is plugged in properly and instead called you angrily because something isn't working.  

At that point you have a choice, you could yell at them as well or you could walk away calm yourself and then 'show' them what was wrong.  Most people are then embarrassed by their mistake and apologize.

**Comment by tkapa** [comment]
Sometimes it's hard to recognise a time to apply this in the moment. And sometimes you're forced to just keep an eye on the person tailgating you while you mentally tell them to stop.

Everyone has their own way of dealing with tension, and it helps to find yours.

**Comment by jakebayliss** [comment]
Great tip but I only think this is applicable to reacting to emails

**Comment by Tanya-Leahy** [comment]
Looking at the problem only and taking the emotion out of it can give a clear perspective of why a problem has arisen and what is the solution. Walking away and going to do something else to help clear your thoughts I find helps before tackling the issue.

**Comment by ncn-ssw** [comment]
The default reaction should be to delay sending emotional emails. In most cases, the sender did not intend any offense or may be having a bad day themselves.

However, this should not prevent you from expressing disapproval of poor or inconsiderate behavior. Such criticism should always be targeted to the specific actions they've taken or words spoken, and quickly expressed. Failure to do so may see you become someone's punching bag for whenever they are having a bad day.

**Comment by joshbermanssw** [comment]
I've read some of these and i +1 what Jean said. Just removing yourself from the environment is often the most effective approach

**Comment by steven0x51** [comment]
Great technique

**Comment by isaaclombardssw** [comment]
Especially for written communication where tone / context isn't as straightforward, can help to keep this in mind.

**Comment by Levijj22** [comment]
Interesting - but i don't think this works in all situations. 
Sometimes the issue cant wait 30mins. Sometimes you can't walk away. 

IMO - Its usually best to ask yourself (or that person) why they are acting the way they are. Usually they are having a bad day, or are dealing with something. This is where empathy is important.

**Comment by uly1** [comment]
I usually need to deal with situations as they happen (especially in person) so I've found just taking a couple of breaths before talking is usually enough to alleviate most of the kneejerk reaction

**Comment by Ben0189** [comment]
Great tips, step away but i will also reflect on why i feeling angry or annoyed. By understanding the cause you're less likely to feel this way in the future.

**Comment by TobyChurches** [comment]
Taking a minute to critically think about the situation before you reply is always beneficial. Not only is this just good for me, it also gives the other person some time leading to a much more constructive conversation.

**Comment by Calinator444** [comment]
> Any stories from when you were frustrated and reacted immediately?

Not that I'd want to talk about to be honest. The result is usually embarrassing.

> Any other tips to avoid immediately reacting?

Another good approach to breaking bad habits is to add rewards and milestones for breaking that habit. E.g. a smoker who's gone 5 days without a cigarette might reward themselves by buying an extra cup of coffee. Human fundamentally fundamentally aren't good at prioritizing long term goals over short term rewards, but you can trick your brain.

**Comment by awaltos** [comment]
I like being aware of the potential coupling between impulse and behaviour. It might not always be possible to separate the two or put 30-60 mins between them but being conscious about the two things may give us the chance to think more rationally about a solution.

**Comment by edkranz** [comment]
if i ever feel angry, i'm probably just hungry. go eat something.

**Comment by drwharris** [comment]
I have used this in my roles in Junior sport for 15 odd years. We instruct parents to follow the 24 hour rule. That is that any concerns you have with court time, decision made between player & coach, etc, you should write down your concerns, sit on them for 24 hours and then ask to meet.

**Comment by bettybondoc** [comment]
❌ Reacting
✅ Responding

Normally, when I receive a very frustrating email, I do it in this order:
1. React (to myself, in person) 😤
2. Draft an email (all the nasty things I want to say) 🤬
3. Do something else (to calm down) 🧘🏻‍♀️
4. Cringe at my first draft (you kiss your mother with that mouth?) 🫣
5. Rewrite my draft ✍️
6. Get someone cool-headed to check it for me (said person chops half of it) 😬
7. Send it 🚀

So, if I take too long to respond, it either means I'm really busy, or I'm still processing my reaction internally.

Also, this is why I can't work in Sales.

![image](https://github.com/user-attachments/assets/cbc7f59e-f2a9-467a-ad6f-eaea8e4e7247)
**Figure: Processing Internally**

**Comment by jeoffreyfischer** [comment]
Another thing is to talk to a middle man - someone who can give unbiased advice in the situation

**Comment by BenDunkerleySSW** [comment]
I think this advice is great (in the right context). People often don't realise how much their judgement is affected by emotions - especially negative ones.

**Comment by jaydenalchin** [comment]
An overnight pause is not always feasible! But this is a great technique that puts into words a habit I built without really thinking about it. Write down your knee-jerk thoughts and feelings, come back to them later.

**Comment by Dhruv-0987** [comment]
Any stories from when you were frustrated and reacted immediately?
Not every issue can wait 30 mins when you know are absolutely right and you have spent time learning something to never be wrong then its frustrating

**Comment by ricksu978** [comment]
A tips to handle negative clients.

Keep your partner's photo on your phone lock screen

Whenever you feel discouraged or down

<details>
   <summary>Pick up your phone, look at your partner's photo and say to yourself (👇👇👇)</summary>

# If I can handle him/her, I can handle ANYTHING!
</details>

**Reply by bettybondoc** [reply]
HAHAHAHA it started sweet 🤣

**Reply by jimmy1992abc** [reply]
A tips to handle negative partner.

Keep your client's photo on your phone lock screen

Whenever you feel discouraged or down

Pick up your phone, look at your client's photo and say to yourself

         It is not too bad...

**Comment by chrisschultzssw** [comment]
> Any stories from when you were frustrated and reacted immediately?

Not that I can think of.

> Any other tips to avoid immediately reacting?

As others have said, some things can't wait 30-60 mins or overnight - so you might need a quick way to calm down, e.g. take some deep breaths.

**Comment by amankumarrr** [comment]
A principle I have always adhered to is never making decisions when I'm emotionally driven or agitated. I always wait until I've calmed down. It's important not to react without thinking. It's challenging, but it usually helps.

**Comment by jimmy1992abc** [comment]
Each person has certain aspects of their character that, when triggered, can easily lead them to give in to urges. When this happens, I tend to react negatively, and it's hard to break that habit. 

I think the best way to avoid reacting immediately is to put yourself in the other person's shoes. After spending 15 minutes pretending to be them and reconstructing the scenario in your mind, you're already halfway to success.

**Comment by christoment** [comment]
Good tip! Can potentially save people from long-term disaster by acting impulsively

**Comment by AntPolkanov** [comment]
📝 Written communication is the easiest to handle - you have the opportunity to cool down while writing a message, and you can carefully consider each word.

🗣️ Verbal communication is different - you may not notice how the heat goes up. The tone, pitch, and gestures start playing crucial role in how the other person perceives you. In my experience, focusing on the problem helps to cool down emotions.

**Comment by sethdaily** [comment]
Eating ice cream can help to cool down

**Comment by Marxoz** [comment]
Forcing yourself to listen even though you are not agreeing is important. If you try and understand the other person's point or opinion, the changes of acting negatively is less as you will have a better understanding of where they are coming from.

Pausing the conversation to tell a funny or positive story or change the subject for a while is also a good technique to calm things down.

**Comment by calumjs** [comment]
>Any other tips to avoid immediately reacting?

In the grand tapestry of the cosmos, our existence is but a fleeting moment, a mere blink in the eye of eternity. Consider these mind-bending facts:

1. Vast Distances: Light from the nearest star system, Alpha Centauri, takes over 4 years to reach us. The Milky Way galaxy alone spans 100,000 light-years.
2. Incomprehensible Numbers: There are an estimated 100 billion galaxies in the observable universe, each containing billions of stars.
3. Time Scale: The universe is approximately 13.8 billion years old. If this time were compressed into a single Earth year, human civilization would occupy only the last few seconds of December 31st.
4. Cosmic Insignificance: Earth is a tiny speck orbiting an average star in a spiral arm of an ordinary galaxy. We are cosmically insignificant in terms of size and location.
5. Temporal Impermanence: The average human lifespan is about 79 years. Compared to geological time scales measured in millions of years, our lives are infinitesimally short.
6. Quantum Uncertainty: At the subatomic level, particles pop in and out of existence constantly. Our solid reality is built upon a foundation of quantum uncertainty.
7. Cosmic Cycles: Stars are born and die over billions of years. The atoms in our bodies were forged in the hearts of long-dead stars.
![image](https://github.com/user-attachments/assets/582f6f6a-1f51-466e-92fb-08c2c0bc098c)

**Reply by duanxinhuan** [reply]
We might still exist after death.
Life is more like a prison than a 'life'
And your friend in prison are the best friends, that's why we should love each other and be calm

**Reply by jimmidier** [reply]
So next time I should walk out of the room.. to find this post and re-read the astronomical facts

**Comment by duanxinhuan** [comment]
I have been having anger management issues when I was younger, I'm still not too old.

Manage emotion is easy and hard, the more you passionate about your work the more you get angry easier.

For me, the way to get it better is to be more understanding, and see the other people's pain, when we know each other's pain we will be on each other's side.

Or instead, if it's too hard to deal with, just play like a game, it also helps me a lot when I imagine it's a video game and I want to be top ranker, lol.

**Reply by duanxinhuan** [reply]
The hardest part is how to deal with some one I don't know before.
My friend suggests me to treat everybody as if he/she is an idiot at the first place, I don't know if I should follow it.

**Comment by ozair-ashfaque** [comment]
It's a good idea to decouple the urge from the behavior.

**Comment by stephanfakossw** [comment]
Good tip :-)

**Comment by Aibono1225** [comment]
Good concept, I'll try it out when I have to face this kind of situation

**Comment by jerryluossw** [comment]
Great rule, I've known similar for many years (Like never make a decision when you're angry), but it's very difficult to do that every time.

**Comment by jimmidier** [comment]
Stepping away is useful to calm down. Other times when immediate reaction is required, react before feeling the anger. Emotions can be blocked if not thinking about it, which needs some practice but is helpful to build a strong mind

**Comment by jernejk** [comment]
> Any stories from when you were frustrated and reacted immediately?

When people want absolute answers to questions that are highly sensitive to context and they refuse hard evidence that the question does not have an absolute "yes/no" answer.

For example, "Is .NET upgrade tool good for .NET 8 migration?" is generally true, but they get upset because it's not always true.

> Any other tips to avoid immediately reacting?

If it's in person, asking for a break to continue later or going for a toilet break is a good idea. Sometimes, checking with yourself if you care about the thing can be very productive, because sometimes we are mad for things we don't actually care about.

In the long term, it's important to learn the art of letting things go, which is very different from suppressing emotions. Letting things go clears your mind while suppressing emotions clouds it.

**Comment by babakamyljanovssw** [comment]
Valuable concept. I agree that making decisions under anger emotions can lead to bad outcomes. Will try to use tip on both work and personal life

**Comment by AnkyChe** [comment]
Valuable advice, I'll try it. Although sometimes it might be hard.

**Comment by KristenHu** [comment]
It's a great method to avoid becoming angry and making mistakes!

**Comment by tino-liu** [comment]
This is valuable when connecting with people. This is also why having that conversation after calming down is so important.

**Comment by vladislav-kir** [comment]
> Any other tips to avoid immediately reacting?

If I feel upset by something, I try to take a short break and write down how I am feeling. That way I can look at my emotions in the third person, and maybe see a better way to handle the situation.

**Comment by PennyWalker** [comment]
This is definitely a great way to deal with texts or emails, it's harder in person. Something I've found works for me in person is to let the person know I'm going away to think about it, we can agree to disagree and we can come back to it. That way you can de-escalate an argument without picking a fight by just leaving. This of course only works for people who do not get more angry over time... I defuse quite quickly, but I know people who get more upset after the fact.

**Comment by MattParkerDev** [comment]
It is very important to be aware of and regulate your emotions especially in client facing situations.

**Comment by danielmackay** [comment]
> Any stories from when you were frustrated and reacted immediately?

Not off the top of my head.

> Any other tips to avoid immediately reacting?

If I notice I'm getting worked up I like to write down my thoughts.  This could be in a draft email, or alternatively some private notes.  I'll then review these another day while being calm and double check if the message is worded correctly, or even if it needs to be sent at all!
